test/e2e/run_userns_test.go

```diff
@@ -89,6 +89,14 @@ var _ = Describe("Podman UserNS support", func() {
 		Expect(ok).To(BeTrue())
 	})
 
+	It("podman --userns=keep-id --user root:root", func() {
+		session := podmanTest.Podman([]string{"run", "--userns=keep-id", "--user", "root:root", "alpine", "id", "-u"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		ok, _ := session.GrepString("0")
```

Comment:
Eek, could you make this `ToEqual("0")`? UID 1000, the typical user ID, also contains the string "0"

Reply:
Fixed